### ISSUE TITLE
Accessibility of the Home page

### ISSUE THREAD

**Issue by bl-aire** [jupyterhub/jupyterhub#4262]
### Details from report:

Based on evaluation using the WAVE accessibility tool, the following are responsible for accessibility issues on the Login page:

- **Language missing or invalid**, **Missing first level heading** and **Noscript element**: These are accessibility issues that are common between the login page and home page #4260 so they can be fixed the same way.

- **Very low contrast**: The contrast between the navigation items (Home, Token and username) and their container's background colour is too low so it won't be legible for people with low vision or colour deficiencies except the text's shade is made darker.

- **Aria button**: This means element (other than a button) has `role="button"`. Elements with `role="button"` must function like native buttons. These should typically be replaced with true button elements or one should ensure the element is navigable, can be activated with Enter or Space key presses (key detection is typically necessary), and provides a visible focus indicator.

### Report:

<img width="1280" alt="Screenshot 2022-12-09 at 14 36 02" src="https://user-images.githubusercontent.com/84882370/206853307-51245a46-ddb6-4dff-8bfe-e89bfcb0aee8.png">

@minrk

**Comment by minrk** [comment]
Let's change the link color to `#222`, which should be high enough contrast. Let me know if you have any questions about finding that in the LESS.

Let's try making our button links button _elements_, and see what happens.

**Comment by bl-aire** [comment]
Okay. 
I wil do that

**Comment by bl-aire** [comment]
Hello @minrk ,

I have a question regarding the pages I am evaluating. I have been able to evaluate pages like:
- Home
- Login
- Token

But I haven't figured out how to evaluate templates like:

- **Spawn_pending**: this displays when the **Start my server** is clicked but stays for a couple of seconds so I haven't been able to access it with wave.
- **Oath** , **not_running**: the former should display when a third-party application wants access to data associated with a JupyterHub account while the latter should display when attempts to start the server fails but I don't know how to recreat these yet.
- **Logout**: this is not displaying on my instance when I logout.


How can I have these pages display on my local instance of JupyterHub?

I considered creating an issue for questions related to the project so they can be tracked easily. If that's okay, I can create it when you approve.

**Comment by minrk** [comment]
Spawn pending can be tested more easily by running a custom spawner that delays start:

```python
import asyncio
from jupyterhub.spawner import SimpleLocalProcessSpawner

class SlowSpawner(SimpleLocalProcessSpawner):
    async def start(self):
        await asyncio.sleep(10)
        return await super().start()

c.JupyterHub.spawner_class = SlowSpawner
```

The `not_running` page can be tested by directly visiting `http://localhost:8000/hub/spawn-pending/your-username` when your server is not running.


The oauth page can be tested by running jupyterhub in the `examples/service-whoami` directory, and clicking the `Services > whoami-oauth` link:

<img width="542" alt="Screenshot 2022-12-14 at 13 41 18" src="https://user-images.githubusercontent.com/151929/207598039-849e0d59-c2b7-4e0a-9ad8-62f5b6d8d809.png">


Perhaps we need to explore better ways to get fully rendered pages without needing to recreate the full environment. Maybe the best way to do it is to export HTML snapshots from our selenium tests, I'm not sure.

**Comment by bl-aire** [comment]
Thank you. I will evaluate them.

> Perhaps we need to explore better ways to get fully rendered pages without needing to recreate the full environment. Maybe the best way to do it is to export HTML snapshots from our selenium tests, I'm not sure.

Personally, I haven't tried this but I am researching on it. Hopefully I learn something new I can implement
